subsys/bluetooth/host/hci_core.c

```diff
@@ -1248,7 +1248,7 @@ static inline bool rpa_timeout_valid_check(void)
 #if defined(CONFIG_BT_PRIVACY)
 	/* Check if create conn timeout will happen before RPA timeout. */
 	return k_delayed_work_remaining_get(&bt_dev.rpa_update) >
-	       K_SECONDS(CONFIG_BT_CREATE_CONN_TIMEOUT);
+	       bt_dev.create_param.timeout;
```

Comment:
What if the user passed 0, in which case the default is used but not copied to `bt_dev.create_param.timeout`?

Reply:
Defaults are now resolved in create_param_setup()